kinde_sdk/management/management_client.py

```
@@ -366,238 +84,412 @@ def __init__(self, domain: str, client_id: str, client_secret: str):
+            Use :meth:`client.organizations.get_organizations()` instead.
+        
+        For full documentation and parameters, see OrganizationsApi.get_organizations()
+        """
+        warnings.warn(
+            "get_organizations() is deprecated. Use client.organizations.get_organizations() instead.",
+            DeprecationWarning,
+            stacklevel=2
+        )
+        return self.organizations.get_organizations(**kwargs)
+    
+    def get_organization(self, code: str = None, **kwargs):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)